stdlib/random.pyi

```diff
@@ -7,6 +7,7 @@ from typing import Any, NoReturn, Tuple, TypeVar
 _T = TypeVar("_T")
 
 class Random(_random.Random):
+    VERSION: int
```

Comment:
The version number of random.py hasn't changed since 2007, but representing it as an `int` rather than a literal seems consistent with other modules like pickle: https://github.com/python/typeshed/blob/master/stdlib/pickle.pyi#L5

Reply:
```suggestion
    VERSION: ClassVar[int]
```

Reply:
I now think it's fine to keep `VERSION`, for consistency with pickle and other similar modules.